src/relayserver/RelayServer.ts

```diff
@@ -96,7 +96,8 @@ export interface RelayServerParams {
   readonly web3provider: provider
   readonly keyManager: KeyManager
   readonly contractInteractor: ContractInteractor
-  readonly hubAddress: PrefixedHexString
+  readonly hubAddress: Address
+  readonly trustedPaymasters?: Address[]
```

Comment:
no new tests for trustedPaymasters. separate issue?

Reply:
There is a test

Reply:
where is it? you added a `trustedPaymasters` parameter to RelayServer, but the string doesn't appear in any test.